src/applications/disability-benefits/all-claims/pages/uploadForm4192.jsx

```diff
@@ -0,0 +1,62 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+
+import { DocumentDescription } from '../content/uploadPtsdDocuments';
+
+const FIFTY_MB = 52428800;
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-4192',
+  'ui:description': DocumentDescription,
+  uploaded4192: fileUploadUI('', {
```

Comment:
I mentioned in another PR that I was thinking about making the names for these schema properties more consistent. I like this name. It's direct and unambiguous. :smile:

Reply:
Yeah, I realized that I was trying to get answers internally about what we should be calling a form and realized that already had a very obvious answer!